src/pages/guides/sellers/profile-company.md

```diff
@@ -4,11 +4,11 @@ title: Company profile
 
 # Company profile
 
-Your company profile provides the information about your business and brand that appears in your Commerce Marketplace listing. During the initial interview, the Company Profile page appears after you choose the “Business” type of account. In your account, the Company Profile is located on the Developer Portal tab under My Information.
+Your company profile provides the information about your business and brand that appears in your Commerce Marketplace listing. During the initial interview, the Company Profile page appears after you choose the "Business" type of account. In your account, the Company Profile is located on the Developer Portal tab under My Information.
 
 ![Company profile](_images/developer-portal-company-profile.png)
 
-The following instructions walk you through the process of completing your company profile. All fields are required, unless marked “Optional”.
+The following instructions walk you through the process of completing your company profile. All fields are required, unless marked "Optional".
 
 ## Step 1: Before you begin
```

Comment:
Step 2 and 5 have changed a bit and we have different structure: 

![image](https://github.com/AdobeDocs/commerce-marketplace/assets/44397551/7c471929-3a37-46bb-a7e1-92f11f591de3)

![image](https://github.com/AdobeDocs/commerce-marketplace/assets/44397551/9e039ffc-2727-4f57-88e9-fe48959995a5)